src/java/org/apache/cassandra/config/Config.java

```diff
@@ -392,19 +454,28 @@
     /**
      * Size of the CQL prepared statements cache in MB.
      * Defaults to 1/256th of the heap size or 10MB, whichever is greater.
+     *
      */
-    public Long prepared_statements_cache_size_mb = null;
+    @Replaces(oldName = "prepared_statements_cache_size_mb", converter = Converter.MegabytesDataStorageConverter.class, deprecated = true)
+    public DataStorage prepared_statements_cache_size = null;
+
+    @Replaces(oldName = "enable_user_defined_functions", deprecated = true)
```

Comment:
There is still a reference to `enable_user_defined_functions` in `doc/source/cql/functions.rst` that we should probably change to the new name.